gosnappi/readme.md

```diff
@@ -0,0 +1,82 @@
+# ![snappi](../snappi-logo.png)
+
+[![license](https://img.shields.io/badge/license-MIT-green.svg)](https://en.wikipedia.org/wiki/MIT_License)
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+[![Build](https://github.com/open-traffic-generator/snappi/workflows/Build/badge.svg)](https://github.com/open-traffic-generator/snappi/actions)
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/open-traffic-generator/snappi.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/open-traffic-generator/snappi/alerts/)
+
+Test scripts written in `gosnappi`, an auto-generated Go SDK, can be executed against any traffic generator conforming to [Open Traffic Generator API](https://github.com/open-traffic-generator/models).
+
+[Ixia-c](https://github.com/open-traffic-generator/ixia-c) is one such reference implementation of Open Traffic Generator API.
+
+> The repository is under active development and is subject to updates. All efforts will be made to keep the updates backwards compatible.
+
+## Install on a client
+
+```sh
+go get github.com/open-traffic-generator/snappi/gosnappi
+```
+
+## Start scripting
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+)
+
+func main() {
+	// Create a new API object, A hook to create and apply the configuration on a traffic generator
+	api := gosnappi.NewApi()
+
+	// Set the communication medium to either GRPC or HTTP
+	grpc := api.NewGrpcTransport()
+	grpc.SetLocation("<grpc server ip>:<port>")
```

Comment:
No need to have grpc variable here. Moreover, let's use an actual ip and port (e.g. 127.0.0.1:50071)